portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,36 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Random;
+
+import com.google.gson.Gson;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world! Using this provided class!</h1>");
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {  
+    
+    ArrayList<String> messages = new ArrayList<String>(); 
+    messages.add("Hello!");
+    messages.add("Yo!");
+    messages.add("Whats good?");
+    messages.add("Aye!");
+    messages.add("My friend!");
+
+    Random rand = new Random();
+    int i = rand.nextInt(4);
```

Comment:
It looks like this code can only actually use the first 4 elements of the array above:

https://www.geeksforgeeks.org/java-util-random-nextint-java/

By setting it to 4, you're saying a random number between 0 (inclusive) and 4 (exclusive), so the max index is 3. 

To solve this for any case, just use messages.size() instead of 4 here instead.